.github/workflows/host-agent-deb-apt.yaml

```diff
@@ -97,134 +72,90 @@ jobs:
-        gpg --armor --export-secret-keys example > example/private-keys/pgp-key-$RELEASE_VERSION.private
-
-    - name: Generating GPG for Release - Creating InRelease
+    - name: Set up aptly
       run: |
-        export GNUPGHOME="$(mktemp -d example/pgpkeys-XXXXXX)"
-        cat example/private-keys/pgp-key-$RELEASE_VERSION.private | gpg --import
-        cat example/repos/$RELEASE_VERSION/apt-repo/dists/stable/Release | gpg --default-key example -abs > example/repos/$RELEASE_VERSION/apt-repo/dists/stable/Release.gpg
-        cat example/repos/$RELEASE_VERSION/apt-repo/dists/stable/Release | gpg --default-key example -abs --clearsign > example/repos/$RELEASE_VERSION/apt-repo/dists/stable/InRelease
-
-  # upload-artifacts:
-  #   runs-on: ubuntu-20.04
-  #   needs: build
-  #   strategy:
-  #     matrix:
-  #       arch:
-  #         - ARM
-  #         - x64
-
-  #   steps:
-    - name: Remove extra files & Upload to Github Pages (install.middleware.io)
+        # Download aptly binary
+        wget https://github.com/aptly-dev/aptly/releases/download/v1.5.0/aptly_1.5.0_linux_amd64.tar.gz
+        tar -xzf aptly_1.5.0_linux_amd64.tar.gz
+        chmod +x aptly_1.5.0_linux_amd64/aptly
+         
+        # Move aptly binary to a directory in PATH
+        sudo mv aptly_1.5.0_linux_amd64/aptly /usr/local/bin/
+
+    
+    - name: Verify aptly version
+      run: aptly version
+
+    # Reference : https://www.aptly.info/doc/
+    - name: Add DEB to MW APT Repo
       run: |
-        mkdir -p example/scripts
-        mkdir -p example/configyamls
-        cp -r configyamls example
-        cp configyamls/all/otel-config.yaml example/otel-config.yaml
-        rm -rf example/pgpkeys-*
-        rm -rf example/generate-release.sh
-        rm -rf example/mw-agent_$RELEASE_VERSION-1_all
-        rm -rf example/mw-agent_$RELEASE_VERSION-1_all.deb
-
-        git clone https://.:${{ secrets.GHCR_TOKEN }}@github.com/middleware-labs/install.middleware.io.git
-        cp -r example/private-keys/* install.middleware.io/private-keys/
-        cp -r example/public-keys/* install.middleware.io/public-keys/
-        cp -r example/repos/* install.middleware.io/repos/
-        cp configyamls/all/otel-config.yaml install.middleware.io/configyamls/all/otel-config.yaml 
-        cp configyamls/nodocker/otel-config.yaml install.middleware.io/configyamls/nodocker/otel-config.yaml 
-        cd install.middleware.io
+        sudo apt-get install -y gnupg
+        git clone https://.:${{ secrets.GHCR_TOKEN }}@github.com/middleware-labs/apt.middleware.io.git
+        gpg --import apt.middleware.io/gpg-keys/mw-agent-apt-private.key
+        gpg --import apt.middleware.io/gpg-keys/mw-agent-apt-public.key
+        
+        APTLY_HOME=$(aptly config show | grep rootDir | cut -d '"' -f 4)
+        echo "APTLY_HOME=$APTLY_HOME" >> $GITHUB_ENV
+
+        echo "aptly conf before:"
+        cat ~/.aptly.conf
+
+        echo "something"
```

Comment:
Remove this `echo` or modify the message being echoed

Reply:
Right, removed the messages, were just required for debugging.